### PR TITLE
Edit alt methods guide

### PR DESCRIPTION
**Pull request by ltagliaferri**
Edited the alternative installation guide to include all methods

Some initial feedback from Mark:

> Regarding the alternative installation methods thing, I think ultimately we're going to want a general "How to Install Chainguard Enforce" article that combines instructions for installing with chainctl, raw YAML, and a Helm Chart. That makes the most sense to me.
> Semi-related to that, I might advocate for retitling the Onboarding doc to "Getting Started with Chainguard Enforce for Kubernetes" at some point.

Signed-off-by: ltagliaferri

### CODE REVIEW

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/_index.md [hunk 1]
```diff
@@ -10,4 +10,6 @@ images: []
 weight: 001
 ---
 
-Chainguard Enforce is a comprehensive solution for software supply chain risk management.
\ No newline at end of file
+Chainguard Enforce is a comprehensive solution for software supply chain risk management.
+
+Stay up to date on Chainguard Enforce with the [Changelog](https://edu.chainguard.dev/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/changelog/).
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/_index.md:15]: Added the Changelog at the top

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/alternative-installation-methods.md [hunk 2]
```diff
@@ -16,33 +16,124 @@ toc: true
...
+To install with `chainctl`, first authenticate into `chainctl` before running a command.
+
+```sh
+chainctl auth login
+```
+
+With your cluster already set up, you'll install the Chainguard Enforce Agent with this `chainctl` command:
+
+```sh
+chainctl cluster install --group=$GROUP_ID --private --context $CLUSTER
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/alternative-installation-methods.md:44]: Is it right that `--private` flag is not needed for non-private clusters? Does it matter if it is here as a catchall?

**Review by ltagliaferri — COMMENTED** [review]
This is a reference guide edit that can use an extra set of eyes for clarity, style, and errors only. Thanks!

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/alternative-installation-methods.md (hunk 2 above)
Comment (jamonation) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/alternative-installation-methods.md:44]: Hmm, here's what the flag shows in `chainctl cluster install --help`:
> Kubernetes API endpoint isn't publicly accessible. Cannot be used with managed clusters.

I'm pretty sure this means EKS and GKE clusters won't work with this flag.

Comment (jamonation) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/alternative-installation-methods.md:44]: Ah right, further down here on line 76, so AKS on Azure as well.

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/alternative-installation-methods.md [hunk 3]
```diff
@@ -132,45 +227,14 @@ auth:
 EOF
 ```
 
-With the required authentication information in the `values.yaml`, the chart can then be installed using:
+With the required authentication information inclyded in the `values.yaml` file, you can use the Chart to install the Chainguard Enforce Agent.
```
Comment (jamonation) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/alternative-installation-methods.md:230]:
```suggestion
With the required authentication information included in the `values.yaml` file, you can use the Chart to install the Chainguard Enforce Agent.
```

Comment (jamonation) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/alternative-installation-methods.md:230]: typo

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-user-onboarding.md [hunk 4]
```diff
@@ -89,11 +89,11 @@ You should receive output similar to the following.
   \____| |_| |_| /_/   \_\ |___| |_| \_|  \____|   |_|   |_____|
 chainctl: Chainguard Control
 
-GitVersion:    0.1.39
-GitCommit:     98f4b3bc8a1ef111777f797e8248b737643eedc6
+GitVersion:    0.1.44
```
Comment (jamonation) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-user-onboarding.md:92]:
```suggestion
GitVersion:    <semver version>
```

Comment (jamonation) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-user-onboarding.md:92]: Worth updating this with the generic placeholders now, or should we do it across every page in a separate PR?

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-user-onboarding.md [hunk 5]
```diff
@@ -89,11 +89,11 @@ You should receive output similar to the following.
...
+GitCommit:     dabc530b10168315afd1edf7f71df6a7ba1d844b
```
Comment (jamonation) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-user-onboarding.md:93]:
```suggestion
GitCommit:     <commit hash>
```

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-user-onboarding.md [hunk 6]
```diff
@@ -89,11 +89,11 @@ You should receive output similar to the following.
...
 GitTreeState:  clean
-BuildDate:     2022-12-05T14:04:10Z
-GoVersion:     go1.19.3
+BuildDate:     2022-12-14T22:14:33Z
```
Comment (jamonation) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-user-onboarding.md:95]:
```suggestion
BuildDate:     <date here>
```

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-user-onboarding.md [hunk 7]
```diff
@@ -89,11 +89,11 @@ You should receive output similar to the following.
...
+GoVersion:     go1.19.4
```
Comment (jamonation) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-user-onboarding.md:96]:
```suggestion
GoVersion:     <compiler version>
```

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-user-onboarding.md [hunk 8]
```diff
@@ -89,11 +89,11 @@ You should receive output similar to the following.
...
 Compiler:      gc
 Platform:      darwin/arm64
```
Comment (jamonation) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-user-onboarding.md:98]:
```suggestion
Platform:      <your platform>
```

**Review by jamonation — APPROVED** [review]
Nice to have this all laid out in one spot now, looks good to me!

**Pull request merged**